src/PlaywrightSharp.Chromium/Messaging/ConnectionResponse.cs

```diff
@@ -12,7 +12,7 @@ internal class ConnectionResponse
 
         public string Method { get; set; }
 
-        public JsonElement? Params { get; set; }
+        public JsonElement Params { get; set; }

```

Comment:
@Meir017 ?

Reply:
I'm pretty sure a response will either have the `Params` property or the `Result` property. I added a check to see if  `Params` is an object type (ie: event data), otherwise the result is a response for a command

Reply:
anyways, I reverted this